saleor/graphql/product/mutations/products.py

```diff
@@ -334,6 +372,29 @@ class Meta:
         description = 'Updates an existing product.'
         model = models.Product
 
+    @classmethod
+    def clean_sku(cls, product_type, cleaned_input, errors):
+        pass
+
+    @classmethod
+    @transaction.atomic
+    def save(cls, info, instance, cleaned_input):
+        instance.save()
+        if not instance.product_type.has_variants:
+            variant = instance.variants.first()
+            update_fields = []
+            if 'track_inventory' in cleaned_input:
+                variant.track_inventory = cleaned_input.get('track_inventory')
```

Comment:
Here and similarly in the lines below we can just write:
```
variant.track_inventory = cleaned_input['track_inventory']
```
No need to do `get` since at this point we know that this key is present in the dict.